### PR TITLE
Migrate docs from wiki to github

### PR DESCRIPTION
**Pull request by sladyn98**
Migrate the documentation from wiki to github.

### CODE REVIEW

**Comment by sladyn98** [comment]
CC @timja

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,60 @@
+## Changelog
+
+### Version 1.5 (2018-12-11)
+
+-   Compatibility with [Configuration as Code
+    Plugin](https://wiki.jenkins.io/display/JENKINS/Configuration+as+Code+Plugin)
```
Comment (timja) [inline CHANGELOG.md]: update it to the plugins.jenkins.io link
```suggestion
    Plugin](https://plugins.jenkins.io/configuration-as-code)
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+
+### Version 1.0 (Jun 16, 2014)
+
+-   First general release.
+
+### Version 1.0 beta 4 (Oct 01, 2013)
+
+-   Added random UUID into bound file paths.
+-   Factored build wrapper into [Credentials Binding
+    Plugin](https://wiki.jenkins.io/display/JENKINS/Credentials+Binding+Plugin),
```
Comment (timja) [inline CHANGELOG.md]:
```suggestion
    Plugin](https://plugins.jenkins.io/credentials-binding),
```

**Comment by sladyn98** [comment]
> README needs updating to remove wiki link and to include the description section

Yeah working on it :)

File: README.md [hunk 3]
```diff
@@ -26,4 +54,4 @@ To test in a local Jenkins instance
 
     mvn hpi:run
 
-  [wiki]: https://wiki.jenkins-ci.org/display/JENKINS/Plain+Credentials+Plugin
+  [wiki]: https://plugins.jenkins.io/plain-credentials
```
Comment (timja) [inline README.md]: Probably not needed link anymore, you can view docs either on plugin site or GitHub

File: README.md [hunk 4]
```diff
@@ -2,6 +2,34 @@
 
  Allows use of plain strings and files as credentials to be passed to miscellaneous build steps. See also this [plugin's wiki page][wiki]
 
+## Description
+
+This plugin provides two new types of credentials:
+
+-   `FileCredentials` or *Secret file*:
+
+![](https://wiki.jenkins.io/download/attachments/69763266/secretFile.png?version=1&modificationDate=1463752202000&api=v2){.confluence-embedded-image
```
Comment (timja) [inline README.md]: Don’t link to the wiki for images include them in the GitHub repo

File: README.md [hunk 5]
```diff
@@ -2,6 +2,34 @@
...
+![](https://wiki.jenkins.io/download/attachments/69763266/secretText.png?version=1&modificationDate=1463752223000&api=v2){.confluence-embedded-image
+.confluence-content-image-border width="680"}
+
+## Usage
+
+These credentials were originally designed to be used together with the
+[Credentials Binding
+Plugin](https://plugins.jenkins.io/credentials-binding).
+See the
+[wiki](https://wiki.jenkins-ci.org/display/JENKINS/Credentials+Binding+Plugin){.external-link}
```
Comment (timja) [inline README.md]: Link to the plugin site not wiki

Comment (sladyn98) [reply README.md]: Done +1

File: README.md (hunk 3 above)
Comment (sladyn98) [reply README.md]: Yeah removed :+1

File: README.md (hunk 4 above)
Comment (sladyn98) [reply README.md]: Yeah Added them to the repo

File: README.md [hunk 6]
```diff
@@ -2,6 +2,36 @@
 
  Allows use of plain strings and files as credentials to be passed to miscellaneous build steps. See also this [plugin's wiki page][wiki]
 
+## Description
+
+This plugin provides two new types of credentials:
+
+-   `FileCredentials` or *Secret file*:
+
+<p align="center">
```
Comment (timja) [inline README.md]: Use markdown markup for images and I think you need to link with a / at the start of the hyperlink to make it work on the plugin site, I.e /docs not docs

File: README.md [hunk 7]
```diff
@@ -2,6 +2,32 @@
 
  Allows use of plain strings and files as credentials to be passed to miscellaneous build steps. See also this [plugin's wiki page][wiki]
```
Comment (timja) [inline README.md]:
```suggestion
 Allows use of plain strings and files as credentials to be passed to miscellaneous build steps.
```

File: README.md [hunk 8]
```diff
@@ -2,6 +2,32 @@
...
+-   `StringCredentials` or *Secret text*:
+
+![SecretText screenshot 1](/docs/images/secretText.png "SecretText screenshot 1")
+
+## Usage
+
+These credentials were originally designed to be used together with the
+[Credentials Binding
+Plugin](https://plugins.jenkins.io/credentials-binding).
+See the
```
Comment (timja) [inline README.md]:
```suggestion
```

File: README.md [hunk 9]
```diff
@@ -2,6 +2,32 @@
...
+[plugin](https://plugins.jenkins.io/credentials-binding){.external-link}
```
Comment (timja) [inline README.md]:
```suggestion
```

File: README.md [hunk 10]
```diff
@@ -2,6 +2,32 @@
...
+for an example use case.
```
Comment (timja) [inline README.md]:
```suggestion
```

File: README.md [hunk 11]
```diff
@@ -2,6 +2,32 @@
...
+
+However, as described in the [Credentials
+Plugin](https://plugins.jenkins.io/credentials)
+wiki, they could potentially be used by any other plugin that requires
```
Comment (timja) [inline README.md]:
```suggestion
documentation, they could potentially be used by any other plugin that requires
```

File: README.md (hunk 7 above)
Comment (timja) [reply README.md]: (There's no need to link to a wiki page or the plugin site from itself)

File: README.md [hunk 12]
```diff
@@ -1,6 +1,29 @@
...
+
+## Usage
+
+These credentials were originally designed to be used together with the
+[Credentials Binding
+Plugin](https://plugins.jenkins.io/credentials-binding).
+
+However, as described in the [Credentials
+Plugin](https://plugins.jenkins.io/credentials)
+wiki, they could potentially be used by any other plugin that requires
```
Comment (oleg-nenashev) [inline README.md]:
```suggestion
documentation, they could potentially be used by any other plugin that requires
```

File: README.md [hunk 13]
```diff
@@ -1,6 +1,29 @@
...
+
+![SecretFile screenshot 1](/docs/images/secretFile.png "SecretFile screenshot 1")
+
+-   `StringCredentials` or *Secret text*:
+
+![SecretText screenshot 1](/docs/images/secretText.png "SecretText screenshot 1")
+
+## Usage
+
+These credentials were originally designed to be used together with the
```
Comment (oleg-nenashev) [inline README.md]: would be nice to fix the line breaks

Comment (sladyn98) [reply README.md]: Yeah done

**Comment by sladyn98** [comment]
Sorry for the re-review @timja. You can ignore it.

File: README.md [hunk 14]
```diff
@@ -1,6 +1,25 @@
...
+![SecretFile screenshot 1](/docs/images/secretFile.png "SecretFile screenshot 1")
+
+-   `StringCredentials` or *Secret text*:
+
+![SecretText screenshot 1](/docs/images/secretText.png "SecretText screenshot 1")
+
+## Usage
+
+These credentials were originally designed to be used together with the [Credentials Binding Plugin](https://plugins.jenkins.io/credentials-binding).
+See the [plugin](https://plugins.jenkins.io/credentials-binding){.external-link} for an example use case.
```
Comment (timja) [inline README.md]:
```suggestion
See the [plugin](https://plugins.jenkins.io/credentials-binding) for an example use case.
```

**Comment by timja** [comment]
@jeffret-b could you take a look at this please

**Pull request merged**